Src/DFC.App.MatchSkills.Services.ServiceTaxonomy/ServiceTaxonomyRepository.cs

```diff
@@ -1,8 +1,70 @@
-﻿using DFC.App.MatchSkills.Application.ServiceTaxonomy;
+﻿using System;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Sockets;
+using System.Threading.Tasks;
+using AutoMapper;
+using DFC.App.MatchSkills.Application.ServiceTaxonomy;
+using DFC.App.MatchSkills.Services.ServiceTaxonomy.Models;
+using DFC.Personalisation.Common.Net.RestClient;
+using DFC.Personalisation.Domain.Models;
+
 
 namespace DFC.App.MatchSkills.Services.ServiceTaxonomy
 {
     public class ServiceTaxonomyRepository : IServiceTaxonomyReader, IServiceTaxonomySearcher
     {
+        private static  HttpClient _httpClient=default;
+        public ServiceTaxonomyRepository(HttpClient httpClient)
```

Comment:
What's the reason for this taking a HttpClient instead of a RestClient?

Reply:
It will be fixed now with the refactored class.